src/lib/contexts/userContext.js

```diff
@@ -14,7 +14,7 @@ export const UserContextProvider = ({ children }) => {
 
         return { ...prevValue, ...newValue }
       }),
-    [setUserState],
+    [],
```

Comment:
```suggestion
    [setUserState],
```

@alt-a1r `setUserState` is used in the callback so it should be added to the deps

Reply:
yes, but this value will not change, so useCallback don't need to listen it, if I'm not mistaken

Reply:
but exhaustivce deps will throw warning